### DISCUSSION TITLE
Migrate React frontend to SolidJS

### DISCUSSION THREAD

**Discussion by deeprobin** [grafana/grafana#66185] [General]
[SolidJS](https://www.solidjs.com/) is an emerging but also already widely used web frontend framework in production. Like React, it also uses the JSX/TSX syntax - but the rendering is much faster.

I think a migration could be a good idea and not too much effort, because probably the only hurdle in a possible migration would be to switch hooks to Signals.

**Comment by gelicia** [comment]
Hello, thank you for the suggestion, but we have no plans to migrate frameworks at this time.
